main.go

```diff
@@ -24,7 +24,8 @@ import (
 
 func main() {
 	var serverAddr = flag.String("s", stun.DefaultServerAddr, "STUN server address")
-	var localPort = flag.Int("p", 0, "The port on which to bind requests, set to 0 to pick a random port")
+	var localPort = flag.Int("p", 0, "The ip on which to bind requests, set to empty will keep default")
+	var localIP = flag.String("i", "", "")
```

Comment:
Why not have one addreess that takes an address with a port?

Reply:
Because in some cases you only need to specify the port, not the IP?

Reply:
You could still do "192.168.0.10:0" or ":5738" or both etc.